gravitee-apim-portal-webui-next/src/services/api.service.ts

```diff
@@ -42,4 +59,8 @@ export class ApiService {
   details(apiId: string): Observable<Api> {
     return this.http.get<Api>(`${this.configService.baseURL}/apis/${apiId}`);
   }
+
+  content(apiId: string, detailsId: string): Observable<ApiDetails> {
```

Comment:
This should be in the `pageService` and return a `Page` object.

Reply:
Done